toolchain/autoupdate_testdata.py

```diff
@@ -37,11 +38,22 @@ def main() -> None:
         else:
             exit(f"Build mode not found in `bazel-bin` symlink: {link}")
 
+    # TODO: Add proper argument parsing.
+    if "--allow-check-fail" in args:
```

Comment:
Had you considered `--non-fatal-checks` to match the `--config`?

Reply:
Yeah, I've gone back and forth on this. I found this flag name more intuitive when running `autoupdate_testdata`, but using a single name also seems nice, and this name doesn't work so well for the config. If you have a strong opinion I'm happy to defer to that.

Reply:
I have a slight leaning towards the same name because it's niche, so my instinct is keeping the same name in spots would make it clearer that they're closely related. I wouldn't call it a strong opinion though.

Perhaps a different name would help -- maybe `no-abort-on-check` (or `disable-check-abort`, or similar) or `log[ging]-checks`? Mirroring something similar to the `#ifdef`... I don't know if any of those names sound better to you in the various spots.

Reply:
A slight leaning is more than I have, so I've gone with `non-fatal-checks` throughout.